### PR TITLE
refactor(Radio): change label

### PR DESCRIPTION
**Pull request by johnrazeur**
## Type

- Refactor

#### The following changes where made:

The radio component can be use without a displayed label, the aria-label prop have been added to ensure accessibility.

### CODE REVIEW

**Review by BABAK0T0 — CHANGES_REQUESTED** [review]
Better to removes the root label when there is no label and puts the aria-label on the input

File: packages/ui/src/components/Radio/index.tsx [hunk 1]
```diff
@@ -138,7 +140,17 @@ type RadioProps = {
     | 'id'
     | 'name'
     | 'required'
-  >
+  > &
+  (
+    | {
+        'aria-label': string
```
Comment (philibea) [inline packages/ui/src/components/Radio/index.tsx:146]: This is not typed when we do that right ?

Comment (philibea) [reply packages/ui/src/components/Radio/index.tsx:146]: should we use ariaLabel as props when we control usage with children ?

Comment (johnrazeur) [reply packages/ui/src/components/Radio/index.tsx:146]: It's typed, it's an XOR (the XOR custom type break eslint idk why). The aria-label prop can't be used when `children` is provided.

**Review by matthprost — APPROVED** [review]

**Review by BABAK0T0 — APPROVED** [review]

**Pull request merged**